source/includes/_files.md

```diff
@@ -0,0 +1,199 @@
+# Files
+
+On the OpenDataSoft platform, files can be used as data source to create new datasets.
+
+## The file object
+
+> Example file object
+
+```json
+{
+    "file_id": "cheese_data.csv",
+    "url": "odsfile://cheese_data.csv",
+    "filename": "Cheese Data.csv",
+    "properties": {
+        "mimetype": "text/csv"
+    },
+    "created": "2017-06-07T15:16:05.701266+00:00"
+}
+```
+
+The file object contains:
+
+* a unique identifier
+* a URL for use in the platform
+* the file name
+* a set of file properties
+* a creation date
+
+### Attributes
+
+Attribute | Description
+--------- | -----------
+`file_id` <br> *string* | Unique identifier for the file
+`url` <br> *string* | Generated URL for use within the platform
+`filename` <br> *string* | Human readable name of the file
+`properties` <br> *object* | Set of file properties, such as its mimetype
```

Comment:
Should be defined, same as the params json object in the records_errors PR, we need to explain what this object is.